src/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <h1>Today's To Do</h1>
+            <i class="fa-solid fa-arrows-rotate"></i>
+        </header>
+        <form action="">
+          <input type="text" name="list" id="list" placeholder="Add to your list...">
+          <i class="fa-solid fa-arrow-turn-down fa-rotate-90"></i>
+          <!-- <i class="fa-solid fa-arrow-turn-down fa-rotate-270 fa-sm"></i> -->
+        </form>
+        <ul class="list--container">
+            <!-- <li class="list--item">
+                <div>
+                    <input type="checkbox" name="" id="">
+                    <p>wash the dishes</p>
+                </div>
+                <i class="fa-solid fa-ellipsis-vertical"></i>
+            </li>
+            <li class="list--item">
+              <div>
+                  <input type="checkbox" name="" id="">
+                  <p>wash the dishes</p>
+              </div>
+              <i class="fa-solid fa-ellipsis-vertical"></i>
+          </li>
+          <li class="list--item">
+            <div>
+                <input type="checkbox" name="" id="">
+                <p>wash the dishes</p>
+            </div>
+            <i class="fa-solid fa-ellipsis-vertical"></i>
+        </li> -->
```

Comment:
### **OPTIONAL**

- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.